pkg/controller/bundles/bundles_syncer.go

```diff
@@ -3,39 +3,65 @@ package bundles
 import (
 	"context"
 	"fmt"
+	"sync"
 
 	"github.com/go-logr/logr"
 	"github.com/open-cluster-management/leaf-hub-spec-sync/pkg/bundle"
 	"github.com/open-cluster-management/leaf-hub-spec-sync/pkg/controller/helpers"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/client-go/rest"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 // LeafHubBundlesSpecSync syncs bundles spec objects.
 type LeafHubBundlesSpecSync struct {
-	log               logr.Logger
-	k8sClient         client.Client
-	bundleUpdatesChan chan *bundle.ObjectsBundle
+	log                  logr.Logger
+	bundleUpdatesChan    chan *bundle.ObjectsBundle
+	k8sClients           []client.Client
+	clientWorkersJobChan chan *clientWorkerJob
+	clientWorkersWG      sync.WaitGroup
```

Comment:
Completely cosmetic comment, maybe should expand to full name "clientWorkersWaitGroup" since others are as well. 

Feel free to ignore.

Reply:
done